.changeset/shy-moons-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@xstate/vue': minor
+---
+
+New useInterpret & useActor + updated useMachine & useService composables ported from @xstate/react.
```

Comment:
Added new `useInterpret` which is a low-level composable that interprets the `machine` and returns the `service`:

```js
import { useInterpret } from '@xstate/vue';
import { someMachine } from '../path/to/someMachine';

export default defineComponent({
  setup() {
    const state = ref();
    const service = useInterpret(machine, {}, (nextState) => {
      state.value = nextState.value;
    });
    return { service, state };
  }
});
```